src/viewer.js

```diff
@@ -221,9 +251,6 @@ module.exports = class Viewer {
 
     this.controls.reset();
 
-    object.position.x += (object.position.x - center.x);
-    object.position.y += (object.position.y - center.y);
-    object.position.z += (object.position.z - center.z);
```

Comment:
I like the addition of the XYZ axes, but I'm worried about not centering the object anymore. Unfortunately it's not that uncommon to see a model with an origin way off `0,0,0`, and (at least with the rest of the `setContent` logic as it currently exists) that could leave many users in a position where they see nothing at all on the screen initially.

I'm open to suggestions on how to avoid that.

Reply:
Ok.  Bringing it back.